### PR TITLE
Help screen display when no course periods setup

### PR DESCRIPTION
**Pull request by nathanstitt**
screen shot 2015-12-06 at 1 40 42 pm

screen shot 2015-12-06 at 12 39 57 pm

### CODE REVIEW

**Comment by nathanstitt** [comment]
Still needs button at bottom to take them to screen and the desktop image will always have a yellow banner (should change to match course)

File: src/components/cc-dashboard/desktop-image.cjsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <path d="M218.288,319.175c-22.113,-9.634 -18.777,-70.415 -18.777,-70.415l-25.819,0c0,0 2.645,60.781 -18.777,70.415l63.373,0Z" style={fill:'#9a9a9b'} />
+      <path d="M100.929,318.646l171.344,0c2.592,0 4.693,2.103 4.693,4.693c0,2.593 -2.103,4.695 -4.693,4.695l-171.344,0c-2.592,0 -4.695,-2.104 -4.695,-4.695c0,-2.591 2.102,-4.693 4.695,-4.693Z" style={fill:'#9a9a9b'} />
+      <rect x="173.691" y="248.76" width="25.819" height="11.736" style={fill:'#828282'} />
+      <path d="M30.249,0l312.042,0c17.232,0 30.911,13.245 30.911,30.472l0,187.774c0,17.226 -13.28,30.514 -30.515,30.514l-312.174,0c-17.233,0 -30.513,-13.288 -30.513,-30.514l0,-187.774c0,-17.227 13.015,-30.472 30.249,-30.472Z" style={fill:'#9a9a9b'} />
+      <path d="M14.083,40.082l0,25.127l267.028,0l40.149,-25.127l-307.177,0Z"
+        className='banner' />
+      <text className='course-name' x="55.571px" y="55.536px">
+          {course.name}
```
Comment (philschatz) [inline src/components/cc-dashboard/desktop-image.cjsx]: Ooh, fancy ; )

**Comment by nathanstitt** [comment]
Screen Shot 2015-12-06 at 3.20.13 PM

**Comment by nathanstitt** [comment]
image

**Comment by nathanstitt** [comment]
image

**Pull request merged**